pkg/process/net/common_linux.go

```diff
@@ -10,14 +10,15 @@ package net
 
 import (
 	"fmt"
+	sysconfig "github.com/DataDog/datadog-agent/cmd/system-probe/config"
```

Comment:
nit: we should probably separate standard go packages from datadog ones.
```golang
import(
    "fmt"
    "os"

    sysconfig "github.com/DataDog/datadog-agent/cmd/system-probe/config"
)
```

Reply:
Good catch. Normally `gofmt` or `goimports` does this automatically. @usamasaqib Let's make sure you've got that setup.